### PR TITLE
WASM conversions which makes this library browser friendly

### LINKED ISSUE #205: `TypeError: fs.readFileSync is not a function` on getBinary in client-js
Seems like `fs` is not part of the package.json, this might be the issue

### LINKED ISSUE #208: [client-js] Wasms conversion so they can be bundled as ts modules
Basically convert them to base64 strings and save as ts files so we won't need to use node-js `fs` module which make it work exclusively in the node-js environment.

### PR DESCRIPTION
**Pull request by hoffmannjan**
So initially we used `fs` module to convert a `wasm` file to byte array that was then consumed by `casper-js-sdk`.
We discussed this with @gyroflaw and found out that during the build process we can convert WASMs to plain ByteArrays and store them inside library as a standard `ts` files which can be important without usage of node-js module. 

So this PR basically removes last node-js dependency and makes it fully browser compatible.

### CODE REVIEW

**Comment by hoffmannjan** [comment]
Sorry that it contain changes initially meant for `1.3` from [this PR](https://github.com/casper-ecosystem/cep-78-enhanced-nft/pull/207) - I can change the target branch if needed to make it easier to read.

File: client-js/scripts/copy-and-convert-wasms.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  /* eslint-disable eslint-comments/no-unlimited-disable */
+  /* eslint-disable */
+  /* prettier-ignore */
+  const base64 = "${base64}";
+  const wasm = new Uint8Array(Buffer.from(base64, 'base64'));
+  export default wasm;
+`;
+
+const runConversion = async () => {
+  await fs.rmSync(PATH_TO, { recursive: true, force: true });
```
Comment (gyroflaw) [inline client-js/scripts/copy-and-convert-wasms.ts:29]: NIT: sync sync methods are used ,the `await`  keyword maybe not required

**Review by gyroflaw — APPROVED** [review]
overall looking good 👍

**Review by gRoussac — APPROVED** [review]
@hoffmannjan seems fine except indeed as Ryo stated runConversion seems to be full synced.

**Pull request merged**